service/resource/service/service.go

```diff
@@ -0,0 +1,63 @@
+package service
```

Comment:
I am a bit (okay more than a bit) unsure about this: My current understanding is, that resources in operatorkit are the ones being reconciled. I added this empty service resource, because this operator is going to reconcile k8s services and keep the referenced IP up to date (and therefore reconciliate it?).

Reply:
Yeah, it is only that the resource will be more about the endpoint and not the service. We add IPs to endpoints. Services are only some piece of the puzzle to do the actual job.

Reply:
Renamed to `endpoint`.